### DISCUSSION TITLE
Advanced search command for searching issues in a repo?

### DISCUSSION THREAD

**Discussion by arunsathiya** [community/community#22618] [New to GitHub]
<p>Is there an advanced search command for searching issues in a repo? I realize one can use <code>repo:owner/name</code> format to search within a repo, but is there a command to add to search just the issues?</p>
<p>My search skills for similar questions on this community are failing me. :slight_smile:</p>

**Comment by mpboom** [comment, marked as answer]
<p>Yeah, you can.</p>
<p>Just add the search query to the end of the input box on <a href="https://github.com/issues/" rel="nofollow noopener">https://github.com/issues/</a> . A query like this:</p>
<pre><code>is:issue is:open repo:atom/atom test
</code></pre>
<p>will search for issues containing the word  <strong>test</strong> in the  <strong>atom/atom</strong> repo.</p>

**Comment by arunsathiya** [comment]
<p>This helps, thank you! </p>
